FirebaseInAppMessaging/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 10.25.0
 - [changed] Removed usages of user defaults API to eliminate required reason impact.
```

Comment:
@ncooke3 Is an addition needed for managing the resources in zip distributions?

Reply:
Yep, good call. Added.